### ISSUE TITLE
Token component needs a11y improvements

### ISSUE THREAD

**Issue by greg-a-smith** [SAP/fundamental-styles#143]
Currently, the Token component (`.fd-token`) is not keyboard accessible.  The "X" appears on the token, but cannot actually accept focus nor be triggered with a keyboard.  Simply adding `role="button"` on a `<span>` is not enough.  The clickable portion (either the "X" or the entire token) needs to be a focusable element, such as a `<button>`.

**Comment by salarenko** [comment]
Token is now keyboard accessible.
